docker-compose.yml

```diff
@@ -1,13 +1,82 @@
+# This docker-compose.yml sets up a deployment of Mastodon's web (Puma), Sidekiq, and streaming containers,
+# as well as the required Postgres and Redis components. Optionally, you can enable Elastic Search and Tor by
+# removing the comments (#) from those blocks.
+
+# You should check the Docker image tag used for the web, sidekiq, and streaming containers to make sure you are running
+# the appropriate version for your deployment. Using `nightly` will deploy the in-development version of Mastodon, which is
+# not considered a stable release. Visit https://github.com/mastodon/mastodon/releases for currently released versions and tags.
+
 version: '3'
 services:
-  db:
+  web:
+    # build:
+    #   context: .
+    #   dockerfile: Dockerfile
+    image: ghcr.io/mastodon/mastodon:nightly
+    env_file: .env.production
```

Comment:
I've actually symlinked, or tried to for .env and .env.production together in my local dev env to all me to run some stuff locally and other parts in docker (though I'm redoing my setup once I can purchase a NUC)